system-addon/content-src/styles/activity-stream-linux.scss

```diff
@@ -1,2 +1,9 @@
 /* This is the linux variant */
+
+$os-arrow-height: 10px;
+$os-arrow-offset-end: 6px;
+$os-arrow-panel-url: 'chrome://global/skin/arrow/panelarrow-vertical-themed.svg';
+$os-arrow-width: 20px;
```

Comment:
If "os-arrow" is the scope/context, a "panel-url" doesn't really make sense? Also, "os-arrow" is a bit generic for this top level scope where these specific values aren't useful for other possible "os-arrow"s.